node-fs-writefile/cp.js

```diff
@@ -0,0 +1,14 @@
+import { writeFile, readFile } from 'node:fs/promises';
+
+const sourceFile = process.argv[2];
+const destinationFile = process.argv[3];
+
+async function copyFile() {
+  try {
+    const fileContent = await readFile(sourceFile, 'utf8');
+    await writeFile(destinationFile, fileContent + '\n');
+  } catch (error) {
+    console.log('error: ', error.message);
+  }
```

Comment:
This works. However, the code you have written works for text files, but not for binary files. Since you provide the 'utf8' encoding when reading the file, you will only be able to successfully read text files. And you convert the contents of the read file to text in line 9 by concatenating a `'\n'` onto it.

To make this work for binary files (like images), all you have to do is remove the 'utf8' encoding and just pass fileContent directly to writeFile!